packages/wdio-sync/src/wrapCommand.ts

```diff
@@ -21,7 +21,7 @@ process.on('WDIO_TIMER', (payload) => {
         while (timers.pop() !== payload.id);
     }
     if (payload.timeout) {
-        elements.forEach(element => { delete element._NOT_FIBER })
+        elements.forEach((element: any) => { delete element._NOT_FIBER })
```

Comment:
is it `WebdriverIO.Element`?

Maybe you can define a type for `elements` and the cast might become redundant

Reply:
No it isn't - inside of the method `wrapCommand` is the only place, where something gets added to the `elements`, which is the following line:
```
elements.add(this)
```
`this`, as per @christian-bromann suggestion would be `WebdriverIO.Client`, so it should be `Set<WebdriverIO.Client>`, but since there's an error on `WebdriverIO.Client` we need to resolve that one first, but I agree - once this is sorted out we can define the type here